File: Assignment No 6/Program4.c
//Write a program which accept number from user and print all odd number up to N.
//Input : 18
//Output : 1 3 5 7 9 11 13

#include<stdio.h>
void OddDisplay(int iNo)
{
	if(iNo<0)
	{
		iNo=-iNo;
	}
	int i=0;

	for(i=1;i<=iNo;i++)
	{
		if(((iNo%i)!=0)||((iNo%2)!=0))
		{
			printf("%d\t",i);
		}
		
	}
}
int main()
{
	int iValue=0;
	printf("Enter the number :\n");
	scanf("%d",&iValue);
	OddDisplay(iValue);
	return 0;
}

File: Assignment No 10/Program4.java
//Problem Statement : Write a Program which accept range from user and return addition of all Even number in between that range
//(range should contain positive numbers only)
//Input : 23 30
//Output :  108 
// Input : -10 2
//Output : Invalid range 
// Input : 90 18
//Output :  Invalid range 
//Input : 10 18
//Output :  70 
import java.io.*;

class Program4
{
	public static int RangeSumEven(int iStart,int iEnd)
	{
		int iSum=0;
		if(iStart>iEnd || iStart < 0)
		{
			System.out.println("Invalid Range");
			return -1;
		}
		for(;iStart<=iEnd;iStart++)
		{
			if(iStart %2==0)
			{
					iSum=iSum+iStart;
			}
		
			
		}
		return iSum;
	}
	public static void main(String args[]) throws Exception
	{
		int iValue1=0,iValue2=0,iRet=0;
		BufferedReader bobj= new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter Starting Point");
		iValue1=Integer.parseInt(bobj.readLine());


		System.out.println("Enter Ending Point");
		iValue2=Integer.parseInt(bobj.readLine());

		iRet=RangeSumEven(iValue1,iValue2);
		System.out.println("Addition is "+iRet);
	}
}